pom.xml

```diff
@@ -272,27 +271,27 @@
 
         <!-- for testing -->
         <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <version>4.13.2</version>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.10.1</version>
```

Comment:
how about moving this version into property, I expect this is the same as in line 282

Reply:
Yup agreed, there are 2 of these.  Once the bots ' renovate or dependabot' run, its better to get one PR than 2.  Will fix that.